Blueprints/BlueprintDefinitions/vs2022/CustomRuntimeFunction-FSharp/template/src/BlueprintBaseName.1/Readme.md

```diff
@@ -51,11 +51,6 @@ Deploy function to AWS Lambda
     dotnet lambda deploy-function
 ```
 
-## Arm64
```

Comment:
This was not needed because anymore. We are targeting .NET 8 and not 7 anymore. And we are using using provided.al2023 not provided.al2. 

Actually this comment was stale because in a later patch of .NET 7 Microsoft lowered the requirements of GLIBC to allow ARM to work on AL2.